### DISCUSSION TITLE
Gsoc selection criteria

### DISCUSSION THREAD

**Discussion by GauravKumar9920** [JdeRobot/jderobot.github.io#235] [General]
Hey I am Gaurav Kumar a TY btech student, I was actively looking towards contributing to this org and did make contributions for the same and was active through out the period, I had really worked hard for Gsoc this year and made significant contribtuions to JDE Robotics and I did learn a lot of new things during the process though.

I had opted for the project number 2 i.e Migration to ROS2 Humble, I even discussed the idea with the mentor @pariaspe and made all the necessary contributions and solved the issues I was asked to, I had drafted the proposal taking in consideration the response given by @pariaspe. 

After everything someone who contributed not so much (being honest) and entered the discussion after proposal submission got selected over me for the project, does the JDE community dosen't value my work and skills ?

**Comment by jmplaza** [comment]
Hi @GauravKumar9920 ,

sorry you have not been selected this time. The  number of GSoC slots is short and the competition is hard. We value your work and skills, and also those from other candidates. Our mentors help all the candidates (you and others) who ask for feedback to improve their proposals. We don't request anything, just suggest things to be refined and improved. 

We are also honest :-) and evaluate of the candidates looking at many indicators: their programming tests, the quality of their proposals, their CV, their contributions to the repositories, their proactivity, their interactions with us.... And then make our fair decision.

Anyway, this is open source organization on a volunteer basis, the main point is to develop open source software which is useful for many people. Feel free to use RoboticsAcademy and contribute.
